runtime/src/accounts_hash.rs

```diff
@@ -994,7 +991,7 @@ impl AccountsHasher {
                     overall_sum = Self::checked_cast_for_capitalization(
                         item.lamports as u128 + overall_sum as u128,
                     );
-                    hashes.write(&item.hash);
+                    hashes.write(&item.hash, &self.dir_for_temp_cache_files);
```

Comment:
we call this 10B times with 10B accounts. i was imagining we would use the path once per 'file', which contains many, many hashes instead of passing it each time. I have several other potential changes to try to batch these writes, but I haven't seen any measurable improvements from batching. But, hash calc IS slow and doing something the slow it seems like the wrong move.